### ISSUE TITLE
Organization of Table in Figma file

### ISSUE THREAD

**Issue by vavalos5** [grommet/hpe-design-system#1137]
Find a way to organize the file in a way that easier to access section. Right now it is a bit all over the place and it would be good to break the content down 

**List of things to work on:**

- [x] Changing the old sorting icons to new icons to the components(need to add outline icons into icon library)

- [x] Figure out whether we need to use auto-layout on components or not in order for headers to continue to work properly as there is now a resizable column that has been included into our designs (this is an exploration) or if constraints are sufficient

- [x] Resizable columns to be added to designs (a way so that they can be turned off and on within the component)

- [x] Update the design examples ( keyboard + mouse interactions)  with the new sorting icons so the flow makes sense

- [x] Adding Batched Items/ multi select from Raja + Guidance

- [ ] Adding Responsive tables from Raja's designs + guidance

- [x] Adding abbreviated header labels from Raja's designs + guidance

- [x] Include new blur color on frozen columns, header and footer to align w/ design system site

- [x] How to use the component (1-4 steps max)

- [x] ToC that directs users to the correct frame of what they're looking for

- [x] Adding new filter to all the designs/examples to align with Greg's designs

- [x] Correct all of the keyboard + mouse interactions with the new abbreviated text, filter, total items below search and include correct spacing

- [x] Include search w/ total items to designs to align with new designs

- [x] Organizing each section so they are in their own frame along with guidance ( search, filter, column groups, components)

- [x] Go through everything again after completed to make sure that nothing is missing or broken (will need other eyes 👁️ on that once I get there

- [x] Include an anatomy for default cells

**Comment by vavalos5** [comment]
I discovered today that I think I'm going to have to put something together that shows how to use the components for table. As I was going back and reorganizing, I felt lost. It's something I want to document so I don't forget.

I'm really liking the new icons on the table. It looks more cohesive than the previous one. I went back and changed the icons for the headers. Also changed it in the mouse and keyboard interactions. 
<img width="195" alt="Screen Shot 2020-10-01 at 3 28 37 PM" src="https://user-images.githubusercontent.com/62117454/94883250-063b9f00-041f-11eb-88eb-134651bad144.png">

**List of things to still work on:**

- [ ] Adding Batched Items/ multi select from Raja + Guidance

- [ ] How to use the component (1-4 steps max)

- [ ] ToC that directs users to the correct frame of what they're looking for

- [ ] Adding new filter to all the designs/examples to align with Greg's designs

- [ ] Include search w/ total items to designs to align with new designs

- [ ] Organizing each section so they are in their own frame along with guidance ( search, filter, column groups, components)
